roundrect/ButtonStyle.swift

```diff
@@ -129,16 +115,13 @@ extension UIButton {
     }
   }
   
-  public convenience init(style: Style, size: Size = .big, theme: Theme? = nil, type: UIButton.ButtonType = .system) {
+  public convenience init(style: Style, size: Size = .big, theme: Theme, type: UIButton.ButtonType = .system) {
```

Comment:
You are introducing some new features here, and changing some public function signature like this one. Any plans for bumping the release number?

Reply:
After this is merged, I will bump the release number